src/app/components/LoadableValue/index.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ *
+ * LoadableValue
+ *
+ */
+import React from 'react';
+import { Tooltip } from '@blueprintjs/core';
+
+interface Props {
+  value: React.ReactNode;
+  loading: boolean;
+  loaderContent: React.ReactNode;
+  tooltip?: React.ReactNode;
+}
+
+export function LoadableValue(props: Props) {
+  if (props.loading) {
+    return (
+      <span className="skeleton-wrapper tw-inline-block tw-whitespace-nowrap tw-overflow-hidden tw-m-0 tw-p-0">
```

Comment:
I think tw- prefix for tailwindcss classes is used only for live dapp, or governance config uses prefix too?
```suggestion
      <span className="skeleton-wrapper inline-block whitespace-nowrap overflow-hidden m-0 p-0">
```

Reply:
yes, need to remove